calculators.md

```diff
@@ -0,0 +1,19 @@
+## Calculators
+
+Sometimes, you may find yourself consistently using a complex model, changing only a few input metrics and checking only
+a few output metrics each time. In this case, it may be more helpful to create a _calculator_ from that model.
+Calculators allow you to isolate a small number of input nodes and output nodes to display as a form rather than a
+model. Any viewer of the space can then enter different numbers for the selected inputs, and the selected outputs,
+computed using the rest of the model's saved inputs, will be displayed.
+
+Here is an [example caluclator](https://www.getguesstimate.com/calculators/70),
+created from [this model](https://www.getguesstimate.com/models/4457):
+
+![Example Calculator](assets/Example Calculator.png)
+
+Calculators are especially useful when a model serves as a template, to be used repeatedly with different inputs, or
+when you want to survey many people about their predictions. To create a calculator, click on the 'calculator' icon
+within the modeling toolbar. Then, you will see a menu that will allow you to create a new calculator. You will be able
+to select which inputs you want to present to the user, and which outputs to display upon calculation.
+
+![New Calculator Form](assets/New Calculator.png)
```

Comment:
I'd probably eventually add information about the following points:

- Points and ranges are valid inputs
- The output can be seen in ranges
- Private/public ness is consistent with model
- These can be shared.
- The result allows you to click on the ? and see more info
- What it means to modify the input when you open the calc in the sidebar (and why you shouldn't save after this)